src/components/course/routes/ExternalCourseEnrollment.jsx

```diff
@@ -98,26 +103,53 @@ const ExternalCourseEnrollment = () => {
                 show={isDuplicateExternalCourseOrder(externalCourseFormSubmissionError)}
                 actions={[
                   <Button as={Hyperlink} target="_blank" destination={externalDashboardUrl}>
-                    Go to dashboard
+                    <FormattedMessage
+                      id="executive.education.external.course.enrollment.page.goToDashboard.cta"
+                      defaultMessage="Go to dashboard"
+                      description="Button text to go to the dashboard"
```

Comment:
Please change the description to `Button that will navigate the learners to learner dashboard`